README.md

```diff
@@ -2,6 +2,8 @@ Most of theses frogs are based on the SVGs of [Twemoji](https://twemoji.twitter.
 
 Have fun using them! 🐸
 
+A website which shows all frog images in the current repo can be found here: https://riesi.github.io/frog_emojis/
```

Comment:
I think we can make the sentence a bit friendlier?

`Here is a website showing all frog images in the current repository:`

Also it makes more sense to put it above the "Have fun using them"